js/gui.js

```diff
@@ -110,12 +110,6 @@ function reset() {
     location.reload();
 }
 
-// On Screen Keys
-$("#leftKey").on("click", keyHandlers.ArrowLeft);
-$("#rightKey").on("click", keyHandlers.ArrowRight);
```

Comment:
Jquery controls were not working ?

Reply:
The problem was that when we hover on the button, an overlay appears on it due to Materialize.css

So we are actually clicking the overlay rather than the actual anchor tag.

But the `onclick` event seems to be working.

Reply:
It's a good workaround but we should fix this bug, as it will cause much damage in future !

Reply:
Any ideas?

I don't think this should cause any problem. There is no extra code redundancy as well.

Reply:
I'll debug, what caused the trouble, will update you :)